src/webchat-ui/components/WebchatUI.tsx

```diff
@@ -1599,7 +1599,9 @@ export class WebchatUI extends React.PureComponent<
 						.some(
 							message =>
 								message.source === "user" &&
-								!(message?.data?._cognigy as any)?.controlCommands,
+								!(message?.data?._cognigy as any)?.controlCommands &&
+								!message?.data?.attachments &&
+								!!message?.text?.trim(),
```

Comment:
Shouldn't the condition check if there is text or attachment. ie. (!!message?.data?.attachments ||			!!message?.text?.trim())